devtools-ci-index.yaml

```diff
@@ -3028,6 +3028,32 @@
             <<: *github_pull_request_defaults
     <<: *contract_tests_provider_template
 
+- job-template:
+    name: 'devtools-contract-test-provider-{consumer}-fabric8-auth'
```

Comment:
was this templated to be able to have additional jobs with different consumers?
or is there a use case where you want to use the same consumer but with different creds?

Reply:
Yes, this template is for multiple different consumers where the creds introduced in this template is specific for the provider (fabric8-auth).

Reply:
thanks for clarifying!